integration/tests/tracer/test_tracer_historical_methods.py

```diff
@@ -0,0 +1,293 @@
+import random
+import typing as tp
+import allure
+import pytest
+from integration.tests.basic.helpers.basic import BaseMixin
+from utils.helpers import wait_condition
+
+CONTRACT_CODE = '6060604052600080fd00a165627a7a72305820e75cae05548a56ec53108e39a532f0644e4b92aa900cc9f2cf98b7ab044539380029'
+DEPLOY_CODE = '60606040523415600e57600080fd5b603580601b6000396000f300' + CONTRACT_CODE
+
+
+@allure.story("Tracer API RPC calls historical methods check")
+class TestTracerRpcCalls(BaseMixin):
+    _contract: tp.Optional[tp.Any] = None
+
+    @pytest.fixture
+    def storage_contract(self) -> tp.Any:
```

Comment:
for me better use scope="class" for deploy contract and another autouse fixture to teardown and don't save contract in class

Reply:
with class scope we need to rollback contract state after every test. for me it is easy to have a new clear contract instance for every new test